### ISSUE TITLE
Unable to run Jest tests with version 3.2.0

### ISSUE THREAD

**Issue by danielsogl** [piotrwitek/typesafe-actions#122]
## Description

I updated to the new release 3.2.0 and facing now some test errors because of a "missing" file.

```bash
FAIL src/Component/ProjectMeasuringCriteria/index.test.tsx
  ● Test suite failed to run

    Cannot find module './action' from 'index.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (node_modules/typesafe-actions/dist/index.js:4:16)
```

Looks like the path to the `actions.d.ts` file is wrong.

## Steps to Reproduce
Create a new project with create-app, use redux and typesafe-actions and run some tests with jest.

## Project Dependencies
- Typesafe-Actions Version: 3.2.0
- TypeScript Version: 3.3.3
- tsconfig.json:

{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "preserve"
  },
  "include": [
    "src"
  ]
}

**Comment by gurkerl83** [comment]
Same here

**Comment by piotrwitek** [comment]
Released V3.2.1 with the fix.
